examples/tfengine/generated/team/cicd/triggers.tf

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ================== Triggers for "prod" environment ==================
+
+resource "google_cloudbuild_trigger" "validate_prod" {
+  provider    = google-beta
+  project     = var.project_id
+  name        = "tf-validate-prod"
+  description = "Terraform validate job triggered on push event."
+
+  included_files = [
+    "terraform/**",
+  ]
+
+  github {
+    owner = "GoogleCloudPlatform"
+    name  = "example"
+    pull_request {
+      branch = "^main$"
+    }
+  }
+
+  filename = "terraform/cicd/configs/tf-validate.yaml"
+
+  substitutions = {
+    _TERRAFORM_ROOT = "terraform"
+    _MANAGED_DIRS   = " devops example-prod-secrets example-prod-networks example-prod-data example-prod-apps"
```

Comment:
extra space at beginning of string

also, we should probably try to read in the list of dirs from a file rather than pass it as a string

Reply:
Removed space.